### DISCUSSION TITLE
Brain segmentation not removing skull

### DISCUSSION THREAD

**Discussion by mattcamp1** [dipy/dipy#1553] [Preprocessing] [answered]
## Description
When performing brain segmentation on clinical MRIs, the skull is not being removed properly.  There are some slight changes and noise removal.  Overall the skull remains intact.

![skull stripping](https://user-images.githubusercontent.com/32101172/41052074-e234568c-6985-11e8-9ec6-51a8c332e691.png)

 I am not sure if I am using the median_otsu function incorrectly or not.  If I am, is there an alternative for skull stripping that anyone is aware of?

## Way to reproduce
OS: Windows-10-10.0.17134-SP0
Python version: Python 3.6.5

DiPy version: 0.14.0
NumPy: 1.14.3
SciPy: 1.1.0
Nibabel: 2.2.1
H5Py: 2.8.0

**Comment by Garyfallidis** [comment]
The median otsu command is only recommended for EPI images (e.g. DWI images) not for T1 images etc.

**Comment by matthew-brett** [comment, marked as answer]
Just to unpack: the Otsu method finds a threshold that [in some sense](https://bic-berkeley.github.io/psych-214-fall-2016/otsu_threshold.html) optimally divides the image pixels into two classes, one class has pixel values above the threshold, and the other has pixel values equal to or below the threshold.  In the case of a brain image, this generally gives you a threshold where the pixels above the threshold are in the brain (and in the skull, for your image), and the pixels below threshold are outside the brain / skull.

Otsu on EPI images works for detecting brain not skull simply because EPI images have very little signal from the skull, so the skull signal is close to signal outside the head.

**Comment by Garyfallidis** [comment]
Yes, EPI images have lower intensities on the skull than the brain and therefore they are easier to segment. In a T1 you can have as high or higher intensities in the skull.

**Comment by arokem** [comment]
I believe the question has been answered. 

Do we need to add something to the documentation to clarify this? For example, [here](https://github.com/dipy/dipy/blob/f2e5ecef63b6fc609bda3b6f581a212ae1ad8a35/dipy/workflows/segment.py#L20-L55)?
